src/calendar-list/index.tsx

```diff
@@ -171,12 +168,10 @@ const CalendarList = (props: CalendarListProps, ref: any) => {
   const scrollToMonth = useCallback((date: XDate | string) => {
     const scrollTo = parseDate(date);
     const diffMonths = Math.round(initialDate?.current?.clone().setDate(1).diffMonths(scrollTo?.clone().setDate(1)));
-    if (diffMonths !== 0) {
-      const scrollAmount = calendarSize * pastScrollRange + diffMonths * calendarSize;
-      
-      // @ts-expect-error
-      list?.current?.scrollToOffset({offset: scrollAmount, animated: animateScroll});
-    }
+    const scrollAmount = calendarSize * pastScrollRange + diffMonths * calendarSize;
```

Comment:
scrolling 0 pixels is still a scroll action and can be heavy.
I think it's better to avoid the scroll all together if the scrollAmount is 0

Reply:
Agree. Fixed